### PR TITLE
Add MILA extensions

### PR DESCRIPTION
**Pull request by JoelForamitti**
- Adopt the structure of https://github.com/MILA-Wien/collectivo-quickstart
- Add all MILA extensions that were previously in the main repositories

### CODE REVIEW

File: collectivo/extensions/mila/direktkredit/setup.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        extension=extension,
+        source="db",
+        content="Hier kannst du deine Direktkredite einsehen und verwalten.",
+        requires_perm="collectivo.direktkredit.user",
+    )
+
+    tile.buttons.set([button])
+
+    # Admin objects
+    # TODO Warning if os environ var is missing
```
Comment (nanu-c) [inline collectivo/extensions/mila/direktkredit/setup.py:50]: maybe solve this todo?

File: docker/keycloak-dev/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM quay.io/keycloak/keycloak:19.0.2
```
Comment (nanu-c) [inline docker/keycloak-dev/Dockerfile:1]: The collectivo repo already uses 21.0.2

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: collectivo/extensions/mila/direktkredit/setup.py (hunk 1 above)
Comment (JoelForamitti) [reply collectivo/extensions/mila/direktkredit/setup.py:50]: done!

File: docker/keycloak-dev/Dockerfile (hunk 2 above)
Comment (JoelForamitti) [reply docker/keycloak-dev/Dockerfile:1]: changed!

**Review by nanu-c — APPROVED** [review]

**Pull request merged**